data_structures/queue/priority_queue.py

```diff
@@ -120,45 +115,41 @@ class ElementPriorityQueue:
     priority(16) > priority(32)
     You can change the priorities as per your need
 
-    >>>EPQ = ElementPriorityQueue()
-    >>>EPQ.enqueue(10)
-    >>>EPQ.enqueue(70)
-    >>>EPQ.enqueue(100)
-    >>>EPQ.enqueue(1)
-    >>>EPQ.enqueue(5)
-    >>>EPQ.enqueue(7)
-    >>>EPQ.enqueue(4)
-    >>>EPQ.enqueue(64)
-    >>>EPQ.enqueue(128)
-    >>>EPQ.print_queue()
+    >>> EPQ = ElementPriorityQueue()
+    >>> EPQ.enqueue(10)
+    >>> EPQ.enqueue(70)
+    >>> EPQ.enqueue(100)
+    >>> EPQ.enqueue(1)
+    >>> EPQ.enqueue(5)
+    >>> EPQ.enqueue(7)
+    >>> EPQ.enqueue(4)
+    >>> EPQ.enqueue(64)
+    >>> EPQ.enqueue(128)
+    >>> EPQ.print_queue()
     [10, 70, 100, 1, 5, 7, 4, 64, 128]
-    >>>print(EPQ.dequeue())
+    >>> EPQ.dequeue()
     1
-    >>>print(EPQ.dequeue())
+    >>> EPQ.dequeue()
     4
-    >>>print(EPQ.dequeue())
+    >>> EPQ.dequeue()
     5
-    >>>print(EPQ.dequeue())
+    >>> EPQ.dequeue()
     7
-    >>>print(EPQ.dequeue())
+    >>> EPQ.dequeue()
     10
-    >>>EPQ.print_queue()
+    >>> EPQ.print_queue()
     [70, 100, 64, 128]
-    >>>print(EPQ.dequeue())
+    >>> EPQ.dequeue()
     64
-    >>>print(EPQ.dequeue())
+    >>> EPQ.dequeue()
     70
-    >>>print(EPQ.dequeue())
+    >>> EPQ.dequeue()
     100
-    >>>print(EPQ.dequeue())
+    >>> EPQ.dequeue()
     128
     >>>print(EPQ.dequeue())
     Traceback (most recent call last):
-              File "Sample.py", line 132, in <module>
-                    print(EPQ.dequeue())
-              File "Sample.py", line 100, in dequeue
-                    raise Exception("Under Flow!")
-            Exception: Under Flow!
+        ...
```

Comment:
```suggestion
        ...
    Exception: Under Flow!
```
We still want to check that our exception has the right type and text.